tmk_core/protocol/vusb/vusb.c

```diff
@@ -28,6 +28,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "vusb.h"
 #include <util/delay.h>
 
+#ifdef RAW_ENABLE
+#    include "raw_hid.h"
+#endif
+
+#if (defined(MOUSE_ENABLE) || defined(EXTRAKEY_ENABLE)) && defined(RAW_ENABLE)
+#    error "can't use mouse/extra key and hid raw at same time on VUSB"
+#endif
```

Comment:
IIRC, this is handled automatically, and should error out anyways, if there are too many endpoints enabled.

Reply:
Should this be handled in makefile? Now this is not implemented in makefile.

Reply:
This test is good as it is now, as it's explicit about the mutual exclusivity and conditional compilation requirement of this code.

Reply:
Thanks, Leave this as it is.

Reply:
Ah, okay, LUFA and ChibiOS has a check for the number of endpoints, but V-USB doesn't.  

This is fine, for now, but is somethign that we would want to change in the near future, @fauxpark?

Reply:
@zvecr 's console PR #8559 includes something like that - perhaps we should pull that out and merge it first to simplify both these PRs.